content/post/2019-06-18.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Getting from flat data a world of relationships to visualize with Gephi"
+authors: ["Mariluz Congosto"]
+date: 2019-06-18
+categories: ["R", "Gephi"]
+tags: ["R", "GDF format", "Gephi", "graphs", "visualization"]
+---
+
+Network analysis offers a perspective of the data that broadens and enriches any investigation. Many times we deal with data in which the elements are related, but we have them in a tabulated format that is difficult to import into network analysis tools.
+
+Relationship data require a definition of nodes and connections. Both parts have different structures and it is not possible to structure them in a single table, at least two would be needed. Data analysis tools define different input formats, one of them is GDF, which is characterized by its simplicity and versatility.
+
+In this session, we will see how we can extract the relationships between elements of a file in CSV format to generate a file in GDF format to work with Gephi.
+
+
+## Introduction
+
+One of the most used network analysis tools is Gephi. It is a desktop application that allows us to analyse a network in depth and visualize it. It provides a set of functions to filter nodes or edges, calculate network parameters, apply different layouts, give size and colour to nodes depending on different attributes.
```

Comment:
Como veo que escribes en inglés británico, sugiero cambiar visualize por visualise.

También sugiero que la última frase se acabe con un "and" o "or", ej: ...layouts OR give size and colour...

Reply:
Ok

Reply:
He vuelto a poner el tag "visualization" con "z" porque ya lo tenemos así en un post anterior, y así queda unificado en el wordcloud de la home.